x-pack/plugins/osquery/public/fleet_integration/osquery_managed_policy_create_import_extension.tsx

```diff
@@ -363,32 +388,34 @@ export const OsqueryManagedPolicyCreateImportExtension = React.memo<
   return (
     <>
       {!editMode ? <DisabledCallout /> : null}
-      {policyAgentsCount === 0 ? (
+      {agentlessPolicyIds?.length ? (
         <>
           <EuiFlexGroup>
             <EuiFlexItem>
               <EuiCallOut title="No agents in the policy" color="warning" iconType="help">
                 <p>
                   {`Fleet has detected that you have not assigned yet any agent to the `}
```

Comment:
not strictly related to your PR, but : ) is there no localization support in Osquery plugin? shouldn't these texts be translatable?